test/hotspot/jtreg/vmTestbase/nsk/share/jpda/DebugeeProcess.java

```diff
@@ -108,12 +108,16 @@ public Process getProcess() {
 
     // --------------------------------------------------- //
 
-    /** Created and return new IOPipe channel to the debugee VM. */
+    /** Created and return new IOPipe channel to the debuggee VM.
```

Comment:
Not your typo but would be nice to fix this in the comment:  "Created" => "Create"